index.html

```diff
@@ -4,10 +4,13 @@
     <title>My Page</title>
     <script type="text/javascript" src="scripts.js"></script>
 </head>
-<body>
+<body onload="add_doggo()">
```

Comment:
Looks good!  Alternately you could load your script tag last in the body instead of the head and add a `add_doggo` call inside the script file itself rather than calling it in the `onload` hook.  One advantage with this is when a script tag gets large it can delay the rest of the page loading if it's in the head, but not if it's in the body.  See [this great post](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup/24070373#24070373).
```suggestion
 <body onload="add_doggo()">
```